### DISCUSSION TITLE
GitHub projects - kanban board -  on mobile

### DISCUSSION THREAD

**Discussion by ShrihariMohan** [community/community#10165] [Mobile]
It will easy to see the tasks right ?

**Comment by Gil82OM** [comment]
OMAR GIL LICONA te envió un correo electrónico con el modo confidencial de Gmail:

[image: Logotipo de Gmail]Re: [github/feedback] GitHub projects - kannan board - on mobile (Discussion #10165) <https://confidential-mail.google.com/msg/ACnmsxEr5DcYVc7vCJk7aHPFybjkc4fz-mmZIzNu2G78lz-cglcx6dn1tRUxKRUIjArLJhumxaThTF3Ukbn5kvbqFNuTV3GnlIP30omeXj6jrsyESj2F3akALOrT1fX4_99Dit7HlgPHr-Zu3aRHRqStZ5a45LXPJa25_3yZMW6skn5kfJgvkb42CqCT9LkpMR_lIdfwMrDkzpA=>

Este mensaje se envió el 15 ene 2022 a las 06:04:52 GMT-8
Puedes abrir el mensaje con el siguiente vínculo, que solo funcionará para ***@***.***

Ver correo electrónico <https://confidential-mail.google.com/msg/ACnmsxEr5DcYVc7vCJk7aHPFybjkc4fz-mmZIzNu2G78lz-cglcx6dn1tRUxKRUIjArLJhumxaThTF3Ukbn5kvbqFNuTV3GnlIP30omeXj6jrsyESj2F3akALOrT1fX4_99Dit7HlgPHr-Zu3aRHRqStZ5a45LXPJa25_3yZMW6skn5kfJgvkb42CqCT9LkpMR_lIdfwMrDkzpA=>

El Modo confidencial de Gmail te da más control sobre los mensajes que envías. Es posible que el remitente haya elegido establecer una fecha de vencimiento, inhabilitar la impresión o el reenvío, o dar seguimiento al acceso a este mensaje. Más información <https://support.google.com/mail/answer/7674059>
Gmail, el correo electrónico de Google
El uso está sujeto a la Política de Privacidad de Google <https://myaccount.google.com/privacypolicy?hl=es-419>
Google LLC, 1600 Amphitheatre Parkway, Mountain View, CA 94043, EE.UU.
Recibiste este mensaje porque alguien te envió un correo electrónico con el modo confidencial de Gmail.
[image: Logotipo de Google]
